public/assets/content/Team/content.json

```diff
@@ -148,7 +148,7 @@
         {
           "id": 15,
           "title": "Soumoydip Halder",
-          "content": "Student -1st Year MCA",
+          "content": "IT Enthusiast | Content Creator",
```

Comment:
Please only change for yourself.

Reply:
I did. The PR contains the commits of mine and two others.

Reply:
Verified